client_test.go

```diff
@@ -22,27 +42,26 @@ func testTOPS(t *testing.T, symbols []string) {
 	if len(result) != len(symbols) {
 		t.Fatalf("Received %v results, expected %v", len(result), len(symbols))
 	}
-
-	// TODO(palpant): Test parsing with a mock http client.
-}
-
-func TestTOPS_OneSymbol(t *testing.T) {
-	testTOPS(t, []string{"SPY"})
 }
 
-func TestTOPS_TwoSymbols(t *testing.T) {
-	testTOPS(t, []string{"SPY", "AAPL"})
```

Comment:
I would move this into the test function that uses it, since variable declarations are package-scope and `testCases` in this case only applies to the one test function.